camunda-community-hub/camunda-platform-7-keycloak
# fix: gunnaraccso/keycloak.server:14.0.0 not found

# Camunda Keycloak Identity Provider Pull Request

The gunnaraccso/keycloak.server:14.0.0 isn't found.

## Description
The example/run/docker folder isn't work, because the gunnaraccso/keycloak.server:14.0.0 isn't found. 

## Additional context
I change to use the  gunnaraccso/keycloak.server:21.1.1, and rewrite some config of camunda.env file.

## Testing your changes
1. Add "127.0.0.1 keycloak" in /etc/hosts
2. Run `docker compose up -d`
3. Open localhost:8080, and enter username/password(camunda/camunda1!), then see the main page.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an existing open issue)

## Checklist:
- [ ] My code adheres to the syntax used by this extension.
- [ ] My pull request requires a change to the documentation.
- [x] I have read the **Camunda Community Hub** documentation
- [x] I have read the **Pull Request Process** documentation
- [ ] I have added or suggested tests to cover my changes suggested in this pull request.
- [ ] All new and existing CI/CD tests passed.
- [x] I will /assign myself this issue, and add the relevant [issue labels] to it if they are not automatically generated by Probot.
- [x] I will tag @celanthe in a new comment on this issue if 30 days have passed since my pull request was opened and I have not received a response from the extension's Maintainer
